src/groupeddataframe/splitapplycombine.jl

```diff
@@ -19,34 +19,31 @@ function gen_groups(idx::Vector{Int})
 end
 
 function _combine_prepare(gd::GroupedDataFrame,
-                          @nospecialize(cs::Union{Pair, Base.Callable,
+                          @nospecialize(combine_operations::Union{Pair, Base.Callable,
                                         ColumnIndex, MultiColumnIndex}...);
                           keepkeys::Bool, ungroup::Bool, copycols::Bool,
                           keeprows::Bool, renamecols::Bool)
     if !ungroup && !keepkeys
         throw(ArgumentError("keepkeys=false when ungroup=false is not allowed"))
     end
 
-    cs_vec = []
-    for p in cs
+    processed_combine_ops = map(combine_operations) do p
```

Comment:
we do not want to use `map` here to avoid excessive specialization. Please leave the `Vector{Any}` as a container for `processed_combine_ops`.